scripts/restore.sh

```diff
@@ -100,7 +100,11 @@ tmux_socket() {
 # tmux server each time.
 cache_tmux_default_command() {
 	local default_shell="$(get_tmux_option "default-shell" "")"
-	export TMUX_DEFAULT_COMMAND="$(get_tmux_option "default-command" "-l $default_shell")"
+	local opt=""
+	if [ "`basename $default_shell`"x == "bash"x ]; then
```

Comment:
Can we use `$(basename $default_shell)` instead of backticks? Btw. what's `x` at the end of the strings used for?

Reply:
> Can we use $(basename $default_shell) instead of backticks? 

Sure.

> Btw. what's x at the end of the strings used for?

It's an old habit to avoid test errors (unary operator issues), if one side is empty or undefined.
Theoretically `default_shell` should never be empty, also the double-quotes should prevent the error as well, but it all depends on the bash version and it doesn't hurt to be on the safe side.

Reply:
If you want I can remove the `x`.